src/components/Navbar.vue

```diff
@@ -17,96 +29,133 @@ export default {
       <v-btn to="/" flat>Home</v-btn>
       <v-btn to="/dashboard" flat>Dashboard</v-btn>
       <v-btn to="/leaderboard" flat>Leaderboard</v-btn>
-      <v-btn class="login-btn" icon="mdi-account-circle" size="small" flat></v-btn>
+
+      <v-menu transition="slide-y-transition" v-if="currentUser">
+        <template v-slot:activator="{ props }">
+          <v-btn
+            class="login-btn"
+            icon="mdi-account-circle"
+            size="small"
+            flat
+            v-bind="props"
+          ></v-btn>
+        </template>
+        <v-list>
+          <v-list-item>
+            <v-list-item-title class="d.flex justify-space-between mb-2">
+              <i
+                aria-hidden="true"
+                class="v-icon notranslate mdi mdi-account-circle mr-4"
+              ></i>
+              <span>{{ currentUser }}</span>
+            </v-list-item-title>
+            <v-list-item-title class="d.flex justify-space-between logout">
+              <span @click="LogOut">
+                <i
+                  aria-hidden="true"
+                  class="v-icon notranslate mdi mdi-logout mr-4"
+                />
+                Log Out
+              </span>
+            </v-list-item-title>
+          </v-list-item>
+        </v-list>
+      </v-menu>
+      <v-btn
+        v-if="!currentUser"
+        class="ma-1 white--text v-btn v-btn--is-elevated bg-black theme--light v-size--default secondary color-white"
+        href="/api/login"
+      >
+        <span class="v-btn__content theme--light">
```

Comment:
Would it be helpful if we made this an anchor tag that links the "/api/login" route?

Reply:
Yes let me add an anchor tag